src/app/containers/ProposalPage/Loadable.tsx

```diff
@@ -0,0 +1,6 @@
+import { lazyLoad } from 'utils/loadable';
+
+export const ProposalPage = lazyLoad(
```

Comment:
I'd like to name the component `GovernancePage`.

Reply:
Sounds good. However, I feel that the title may need to change from Vote to Governance for consistency.

Reply:
Sorry. I was confused.